app/services/search.rb

```diff
@@ -110,6 +117,12 @@ def search_resource_fields
       end
     end
 
+    def sort_results
+      sort_by = sort_columns.include?(@sort_by) ? @sort_by : 'updated_at'
+      sort_order = %w[asc desc].include?(@sort_order) ? @sort_order : 'desc'
+      @query.order!(sort_by => sort_order)
```

Comment:
I guess we can avoid generating arrays on every call to check keys:
```
sort_by = @sort_params[:by].presence || 'updated_at' 
sort_order = @sort_params[:order].presence || 'desc'
@query.order!(sort_by => sort_order)
```

Reply:
Hey Anderson,

Thanks for taking a look. The inclusion check was added in order to defend against non-supported values, so we don't end up with an invalid SQL statement. This surely could be implemented at the Grape level instead, but I'd rather have all the search related stuff in one place, i.e. in this service.